### PR TITLE
DOC: Update documentation 

### PR DESCRIPTION
**Pull request by dipinknair**
- [x] Simplify sphinx cards on landing page
- [x] Conda documentation fix
- [x] Simplify cards in getting started and userguide 
- [x] Remove requirement.txt precommit check
- [x] Align cards

### CODE REVIEW

File: doc/source/Getting_started/index.rst [hunk 1]
```diff
@@ -6,7 +6,8 @@ Getting started
    .. grid-item-card:: About PyAnsys and AEDT
             :link: About
             :link-type: doc
-
+            :margin: 2 2 0 0
+            
             Learn more about PyAnsys and Ansys Electronics Desktop.
```
Comment (PipKat) [inline doc/source/Getting_started/index.rst:11]:
```suggestion
            Learn more about PyAnsys and AEDT.
```
AEDT has already been defined. I'd use it consistently.

File: doc/source/User_guide/index.rst [hunk 2]
```diff
@@ -17,49 +17,49 @@ For additional practical demonstrations, see
             :link-type: doc
             :margin: 2 2 0 0
             
-            This tutorial shows how to launch Electronics Desktop and create a project.
+            How to launch Electronics Desktop and create a project.
```
Comment (PipKat) [inline doc/source/User_guide/index.rst:20]:
```suggestion
            How to launch AEDT and create a project.
```

**Comment by Samuelopez-ansys** [comment]
Hi @dipinknair ,

Is this still a draft? I am going to create a new release now, just in case you want me to wait.

**Review by Samuelopez-ansys — DISMISSED** [review]
LGTM

**Review by Samuelopez-ansys — DISMISSED** [review]

**Review by Samuelopez-ansys — APPROVED** [review]

**Comment by MaxJPRey** [comment]
Thanks @dipinknair for updating the PyAEDT documentation.

**Pull request merged**